docs/content/standards/currency.mdx

```diff
@@ -117,25 +140,19 @@ Regulated coin example
 
 ### `DenyList` object
 
-A `DenyList` shared object created by the system contains addresses that aren't able to use a particular regulated currency. If you have access to the `DenyCap`, then you can use the `sui::coin` module to maintain the list of users. Use the functions `coin::deny_list_v2_add` and `coin::deny_list_v2_remove` functions to add and remove addresses.
+The list of addresses that cannot use a particular regulated coin is held within a system-created `DenyList` shared object. If you have access to the `DenyCap`, then you can use the `coin::deny_list_v2_add` and `coin::deny_list_v2_remove` functions to add and remove addresses.
 
 ### Global pause switch
 
-Regulated currencies include an `allow_global_pause` Boolean field. When set to `true`, the bearer of the `DenyCapV2` object for the currency coin type can use the `coin::deny_list_v2_enable_global_pause` function to pause coin activity indefinitely. Immediately upon the bearer initiating the pause, the network disallows the coin type as input for any transactions. At the start of the next epoch (epochs last ~24 hours), the network additionally disallows all addresses from receiving the coin type.
+Regulated coin objects include an `allow_global_pause` Boolean field. When set to `true`, the bearer of the `DenyCapV2` object for the coin type can use the `coin::deny_list_v2_enable_global_pause` function to pause coin activity indefinitely. Immediately upon the bearer initiating the pause, the network disallows the coin type as input for any transactions. At the start of the next epoch (epochs last approximately 24 hours), the network additionally disallows all addresses from receiving the coin type.
 
-When the bearer of the `DenyCapV2` object for the currency coin type removes the pause using `coin::deny_list_v2_disable_global_pause`, the coins are immediately available to use again as transaction inputs. Addresses cannot receive the coin type, however, until the following epoch.
+When the bearer of the `DenyCapV2` object for the coin type removes the pause using `coin::deny_list_v2_disable_global_pause`, the coins are immediately available to use again as transaction inputs. Addresses cannot receive the coin type, however, until the following epoch.
 
-The global pause functionality does not affect the deny list for the currency. After clearing the pause for the currency, any addresses included in the deny list are still unable to interact with the currency's coin.
+The global pause functionality does not affect the deny list for the coin. After clearing the pause for the coin, any addresses included in the deny list are still unable to interact with the coin.
 
-## `TreasuryCap`
+## Coin metadata
```

Comment:
To not confuse with `CoinMetadata` object I suggest calling it Currency Metadata or use a different term. We're adding a feature to borrow `CoinMetadata` object from `Currency` for backwards compatibility, and then we'll get ourselves into a naming trap.